addons/sales_team/views/crm_team_views.xml

```diff
@@ -25,9 +25,9 @@
                     <div class="oe_button_box" name="button_box"/>
                     <widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
                     <div class="oe_title">
-                        <label for="name" class="oe_edit_only" string="Sales Team"/>
+                        <label for="name" class="oe_edit_only o_text_overflow" string="Sales Team"/>
```

Comment:
You do not need to put the class here, the label is used to display "Sales Team" before the name of the team
![_tmp_screenshot](https://user-images.githubusercontent.com/54316226/94011437-a3eff800-fda7-11ea-9a89-8fe9f1f9a6d8.png)